src/website/shared/templates/base.html

```diff
@@ -1,61 +1,66 @@
-{% load static socialaccount %}
+{% load static socialaccount account %}
 
 <!DOCTYPE html>
 <html lang="en">
   <head>
     <title>{% block title %}Nixpkgs Security Tracker{% endblock %}</title>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-    <link rel="stylesheet" type="text/css" href="/static/style.css" />
+    <script src="https://cdn.tailwindcss.com"></script>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Space+Mono">
```

Comment:
I'm not a fan of those, as it pollutes the templates with non-semantic noise that's hard to read (and make any sense of to begin with if you don't happen to know all of the framework). I won't block on it today because we have to get shit done, but really this is a maintenance liability.

Reply:
@fricklerhandwerk as progress gets made the eventual result is simply a tailwind config and generated css file that applies the style of the site based on much simpler rules and much less inline css. 

As a WIP however, to be able to make adjustments on the fly and discover out various styles that make sense, we start from the CDN + exploratory styles.

Otherwise one has to explore out the html, and manage the complexity of the CSS at the same time and refactoring CSS is futile endeavor.

Put simply, this wouldn't remain this way as the style and widgets present on the site stabilize, but as an early PR that is the first in many, complexity will go up before it gets managed back down during this discovery phase.